test/python/test_cudnn_sdpa_torch_ops.py

```
@@ -0,0 +1,212 @@
+
+from cudnn.sdpa.fwd import torch_op  # noqa: E402  (registers torch.ops.cudnn.sdpa_fwd_ex / sdpa_bwd_ex)
+
+TOL = 2.5e-2  # bf16 rounding at these magnitudes
+
+
+def ref_attention(q, k, v, scale, is_causal=False, bottom_right=False, window_left=-1, sinks=None):
+    """fp32 reference in BHSD. window_left counts VISIBLE tokens including self
+    (the cuDNN diagonal_band_left_bound convention). sinks: (H,) extra softmax
+    logit per query head, contributing no value."""
+    q, k, v = q.float(), k.float(), v.float()
+    B, Hq, Sq, D = q.shape
```

Ruff (0.16.1)

[warning] Unpacked variable `D` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)